Readme improvement

eed3si9n opened the pull request:
This adds some documentation about how to configure bzl-gen-build.

non reviewed (approved):
This looks reasonable to me.

It might have been nice to split the change up between documentation and functionality. From what I can tell the script changes are about clearing and then re-downloading the external files, which we want to do when regenerating the builds. Is that right? Am I missing something?

eed3si9n commented:
Re: script changes

1. The script references `fetch_binary()` but was not defined, since we normally just the freshly built on instead of downing, so I added an implementation of `fetch_binary()` here.
2. The other change about `CACHE_KEY` is similar story. A commented out code references `CACHE_KEY` but it wasn't defined, so I added it to demonstrate what should be used as the cache key, if ppl want to cache some JSON files.
